@ornikar/rollup-config/createRollupConfig.js

```diff
@@ -79,6 +79,7 @@ const createBuildsForPackage = (
         isLinariaEnabledForPlatform &&
           linariaPlugin({
             sourceMap: true,
+            classNameSlug: `${(pkg.ornikar && pkg.ornikar.linariaClassnamePrefix) || packageName}_[title]_[hash]`,
```

Comment:
Ça serait peut être plus clair avec un ternaire genre 
```suggestion
            classNameSlug: `${(pkg.ornikar ? pkg.ornikar.linariaClassnamePrefix : packageName)}_[title]_[hash]`,
```